x-pack/legacy/plugins/infra/common/inventory_models/container/metrics/tsvb/container_diskio_ops.ts

```diff
@@ -22,17 +25,17 @@ export const containerDiskIOOps: InfraMetricModelCreator = (timeField, indexPatt
         {
           field: 'docker.diskio.read.ops',
           id: 'max-diskio-read-ops',
-          type: InfraMetricModelMetricType.max,
+          type: 'max',
```

Comment:
Those enums are managed by GraphQL, as part of cleaning up the tech debt I'm making an effort to move everything to IO-TS types which doesn't support enums, but each of those types have been replaced with a `t.keyOf({ ... })` which restricts which string it can be.